File: frontend/src/app/views/profile/coefficient.ts
export class Coefficient {

  public ID: number;
  public IDRA: number;
  public IDHO: number;
  public value: number;


  constructor(ID: number, IDRA: number, IDHO: number, value: number) {
    this.ID = ID;
    this.IDRA = IDRA;
    this.IDHO = IDHO;
    this.value = value;
  }
}

File: servlet/src/main/java/unicyb/horseracingservice/entity/Coefficient.java
package unicyb.horseracingservice.entity;

import com.fasterxml.jackson.annotation.JsonPropertyOrder;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@JsonPropertyOrder({"ID", "IDRA", "IDHO", "value"})
public class Coefficient {
    private int ID;
    private int IDRA;
    private int IDHO;
    private float value;
}
